sdks/python/apache_beam/runners/portability/fn_api_runner.py

```diff
@@ -1444,9 +1485,12 @@ def _generate_splits_for_testing(self,
         break
     return split_results
 
-  def process_bundle(self, inputs, expected_outputs):
+  def process_bundle(self, inputs, expected_outputs, parallel_uid_counter=None):
     # Unique id for the instruction processing this bundle.
-    BundleManager._uid_counter += 1
+    if parallel_uid_counter:
```

Comment:
Why this change. It seems we don't need to ever pass the uid counter in externally.